src/extension/fixes/missingCompileCommands.ts

```diff
@@ -11,11 +11,13 @@ import type { ProjectUE4 } from "../../project/projectUE4";
 
 import * as console from "../../console";
 
+const CancelRemainingMissingFixesCommand = "Cancel Remaining";
+
 /**
- * 
```

Comment:
woops, my settings automatically trims these trailing whitespaces. I can revert these changes if you want?